storybook/stories/API/chart/Treemap.stories.tsx

```diff
@@ -90,3 +90,21 @@ export const Simple = {
     isAnimationActive: false,
   },
 };
+
+export const WithTooltip = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <Treemap {...args}>
+          <Tooltip />
+        </Treemap>
+      </ResponsiveContainer>
+    );
+  },
```

Comment:
You could add a play function to hover over the chart, and then verify that the tooltip is rendered. Thus such an issue could be auto-detected the next time.